app/build.gradle.kts

```diff
@@ -184,10 +185,14 @@ dependencies {
     testImplementation(libs.compose.test.manifest)
 
     // Mockito
-    testImplementation("org.mockito:mockito-core:3.12.4")
-    testImplementation("org.mockito.kotlin:mockito-kotlin:3.2.0")
-    androidTestImplementation("org.mockito:mockito-android:3.12.4")
-    androidTestImplementation("org.mockito.kotlin:mockito-kotlin:3.2.0")
+    //noinspection GradleDependency
```

Comment:
Question: What does this "//noinspection GradleDependency" means?

Reply:
It is the automatic suggestion of AndroidStudio

Reply:
Thanks for Chatgpt answer, but that doesn't say more than what's written... What is the GradleDependency rule? Why did you have to put it there? Warning suppression is usually a bad thing unless you know **exactly** why you do it...

Reply:
PD: By automatic I mean when changing the hard coded dependencies with AndroidStudio's recommendation.